src/transpose/index.js

```diff
@@ -4,6 +4,17 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
+function transpose(array) {
+    const res = [];
+    for (let j = 0; j < array[0].length; j++) {
+        res[j] = new Array(array.length);
```

Comment:
Creating an array this way is not recommended. See using `Array.from({ length: array.length })`

Reply:
thank you